### PR TITLE
[ICDS Dashboard] Fixed the timestamp issue

### PR DESCRIPTION
**Pull request by challabeehyv**
##### SUMMARY
 Fixed the timestamp issue

##### FEATURE FLAG
No

### CODE REVIEW

**Review by Rohit25negi — APPROVED** [review]
Looks good. good to merge as test case passes

File: custom/icds_reports/reports/governance_apis.py [hunk 1]
```diff
@@ -43,8 +33,9 @@ def base_data(row_data):
             expected_visits=get_value_or_data_not_entered(row_data, 'expected_visits'),
         )
 
+    data_rows = []
     for row in data:
-        config['data'].append(base_data(row))
-    config['data'] = config['data'][int(start):(int(start) + length)]
+        data_rows.append(base_data(row))
+    data_rows = data_rows[int(start):(int(start) + length)]
```
Comment (Rohit25negi) [inline custom/icds_reports/reports/governance_apis.py]: help. Catching it now. Applying the pagination here won't buy us performance improvement. this pagination needs to applied on where you are fetching the data

**Review by Rohit25negi — CHANGES_REQUESTED** [review]
left one performance related comment

File: custom/icds_reports/reports/governance_apis.py (hunk 1 above)
Comment (challabeehyv) [reply custom/icds_reports/reports/governance_apis.py]: yeah.thought so. But saw a few implementations earlier on other reports. so made it like that. will change it

File: custom/icds_reports/reports/governance_apis.py [hunk 2]
```diff
@@ -14,16 +14,7 @@ def get_home_visit_data(start, length, year, month, order, query_filters):
...
-        'data': [],
-        'filter_params': {
-            'start': start,
-            'month': month,
-            'year': year,
-            'count': data.count(),
-            'timestamp': india_now()
-        }
-    }
+    paginated_data = data[int(start):(int(start) + length)]
```
Comment (Rohit25negi) [inline custom/icds_reports/reports/governance_apis.py]: 👍

**Review by Rohit25negi — APPROVED** [review]

**Pull request merged**